subsys/bluetooth/audio/shell/bap.c

```diff
@@ -3649,7 +3655,30 @@ static int cmd_init(const struct shell *sh, size_t argc, char *argv[])
 		return -ENOEXEC;
 	}
 
+	snk_cnt = shell_strtoul(argv[1], 0, &err);
```

Comment:
`err` needs to be initialized before this.

It's also missing a check for `err`

Reply:
I'll fix :) As I said above, I didnt intend to push this atm